src/sonic-config-engine/minigraph.py

```diff
@@ -772,7 +772,7 @@ def enable_internal_bgp_session(bgp_sessions, filename, asic_name):
 # Main functions
 #
 ###############################################################################
-def parse_xml(filename, platform=None, port_config_file=None, asic_name=None):
+def parse_xml(filename, platform=None, port_config_file=None, hwsku_config_file=None, asic_name=None):
```

Comment:
Can we introduce the new optional arg as last one ?
This will help in scenarios, where callers pass optional args by position w/o qualifying with name.

Reply:
Modified as requested!